database-plugins/src/main/java/io/cdap/plugin/DBUtils.java

```diff
@@ -185,7 +190,25 @@ public static DriverCleanup ensureJDBCDriverIsAvailable(Class<? extends Driver>
       int columnSqlPrecision = metadata.getPrecision(i); // total number of digits
       int columnSqlScale = metadata.getScale(i); // digits after the decimal point
       String columnTypeName = metadata.getColumnTypeName(i);
+<<<<<<< HEAD
```

Comment:
These conflicts are due to missing commits from https://github.com/cdapio/hydrator-plugins/pull/1033

@vravish will open the cherrypicks for that PR. After that we should fix and merge this PR

Reply:
cherrypick PR is merged https://github.com/cdapio/hydrator-plugins/pull/1275